### PR TITLE
Bump symfony/process version

### PR DESCRIPTION
**Pull request by Reasno**
It would be great to support a newer version.

### CODE REVIEW

**Comment by freekmurze** [comment]
Seems like the tests are failing for this one.

**Comment by Reasno** [comment]
> Seems like the tests are failing for this one.

fixed the forward-compatibility.

File: src/Engines/Node.php [hunk 1]
```diff
@@ -27,7 +27,13 @@ public function run(string $script): string
 
         file_put_contents($tempFilePath, $script);
 
-        $process = new Process("{$this->nodePath} {$tempFilePath}");
+        // Version 4.2 introduced fromShellCommandline
+        $command = "{$this->nodePath} {$tempFilePath}";
+        if (method_exists(Process::class, 'fromShellCommandline')) {
+            $process = Process::fromShellCommandline($command);
```
Comment (sebastiandedeyne) [inline src/Engines/Node.php]: What does this doe that the constructor doesn't?

Comment (Reasno) [reply src/Engines/Node.php]: The signature of constructor changes in `symfony/process` 5.0. Now it only accepts commands as an array, not a string. It also enforces additional escaping with the array format. Without relying on reflection, I think using `fromShellCommandline` (which takes an unescaped string) is the best way to go.

Comment (sebastiandedeyne) [reply src/Engines/Node.php]: Thanks for the clarification!

**Pull request merged**